src/DatePicker/datePicker.scss

```diff
@@ -1,5 +1,5 @@
 @import '../utils/theme/variables.scss';
-@import 'react-dates/lib/css/_datepicker';
```

Comment:
unrelated. Fixes the fix applied [here](https://github.com/buildo/react-components/pull/1471) as it seems to not work with yarn 1